Jinget.Core.Tests/ExtensionMethods/Collections/IQueryableExtensionsTests.cs

```diff
@@ -98,5 +99,24 @@ public void Should_throw_NullReferenceException_when_collection_is_null()
             IQueryable<TestClass> input = null;
             input.OrderByDynamic("", OrderByDirection.Descending);
         }
+
+        [TestMethod()]
+        public void Should_find_given_property_invariant_case()
```

Comment:
This test did not target the method well. Tests should target the `OrderByDynamic ` method directly. Also we need to design the unit tests based on AAA(Arrange, Act, Assert) pattern. Please refer to `Should_sort_the_list_based_on_given_property` method.